cluster-api-gcp/CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contributing Guidelines
+
+## Build
+
+```bash
+$ cd $GOPATH/src/k8s.io/
+$ git clone git@github.com:kubernetes/kube-deploy.git
+$ cd kube-deploy/cluster-api-gcp/
+$ go build
+```
+
+This will create a binary `cluster-api-gcp` that you can use to manage a GCP cluster.
+
+## Developing
+
+When making changes to the machine controller, it's generally a good idea to delete any existing cluster created with an older version of the cluster-api.
+
+```bash
+$ ./cluster-api-gcp delete
+```
+
+After making changes to the machine controller or the actuator, you need to follow these two steps:
+
+1. Rebuild the machine-controller image. Modify the `machine-controller/Makefile` and change the `PROJECT` to your own GCP project. This is important so you don't overwrite the official image. Also change `machineControllerImage` in `cloud/google/pods.go` to the new image path. Then, rebuild and push the image.
```

Comment:
The version on pods.go and the version in the Makefile need to match if you want the code to use the image you just pushed.

Reply:
I've made it more explicit.